src/main/java/mate/academy/model/Movie.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movies")
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String description;
+
+    public Movie() {
```

Comment:
not necessary to add an empty constructor, you don't use it

Reply:
don`t agree. This default constructor use by hibernate to create object of this class when we read data from DB